### ISSUE TITLE
Nodocumentation to add a mock server for development

### ISSUE THREAD

**Issue by geekychaser** [publiclab/PublicLab.Editor#213]
### Please describe the problem (or idea)

> What happened just before the problem occurred? Or what problem could this idea solve?

#### No documentation to setup mock server on our local system to test the editor.

A new developer faces a lot of difficulty in understanding the workflow as there are no steps to create a mock server for development. So I find it most essential to document, on how to setup a mock server, to attract more contributors.

### Please show us where to look

https://publiclab.org/ ...

### What's your PublicLab.org username?
geekychaser

> This can help us diagnose the issue: 

### Browser, version, and operating system

> Many bugs are related to these -- please help us track it down and reproduce what you're seeing!

****

## Thank you!

Your help makes Public Lab better! We *deeply* appreciate your helping refine and improve this site. 

To learn how to write really great issues, which increases the chances they'll be resolved, see:

https://publiclab.org/wiki/developers#Contributing+for+non-coders

**Comment by geekychaser** [comment]
@jywarren Can I work on this issue?
I also faced a lot of difficulty while setting up a mock server and understanding the workflow. So it will be better if I write down the way to directly overcome the difficulty of creating a mock server.
I will add a basic documentation for starting server and will link to a post on publiclab.org for full detail of code and how to setup.

**Comment by jywarren** [comment]
Hmm, do you think we might explain what's going on in the README, and
suggest using the `plots2` app as the mock server? Otherwise this seems
like a really big project, no?

**Comment by geekychaser** [comment]
@jywarren yes you are right, I will add the steps to connect with plots2 as mock server in README file.
thank you
